book/website/reproducible-research/open/open-access.md

```diff
@@ -4,32 +4,73 @@
 (rr-open-access-whatis)=
 ## What Is Open Access?
 
-One of the most common ways to disseminate research results is by writing a manuscript and publishing it in a journal, conference proceedings or book. For many years those publications were available to the public if purchased through a subscription fee or individually.
+Disseminating and sharing scientific results has been part of research since it was established.
+The first scientific journal, Philosophical Transactions, was established in 1665 and published letters about scientific observations and experimentations.
+The 'commercialisation' of scientific publishing began in the 1940s and for many years publications, such as through a journal, conference proceedings or book were available to the public if purchased through a subscription fee or individually.
+
 However, new knowledge is built by synthesizing current scholarship and then building upon it.
-At the turn of the 21st century, a new movement appeared with a clear objective: make all the research results available to anyone interested in reading it, free of charge by any user, with no technical obstacles such as mandatory registration or login to specific platforms.
-This movement took the name of "open access" [{term}`def<Open Access>`] and established two initial strategies to achieve its final goal: self-archiving and open access publishing.
+At the turn of the 21st century, the Open Access [{term}`def<Open Access>`] movement was established, principally through the Budapest Open Access Initative and the Berlin Declaration on Open Access. 
+A quote from the Budapest Open Access Initative sums up the objective:
+>"We mean its free availability on the public internet, permitting any users to read, download, copy, distribute, print, search, or link to the full texts of these articles...the only constraint on reproduction and distribution, and the only role for copyright in this domain, should be to give authors control over the integrity of their work and the right to be properly acknowledged and cited"
+
+It has been estimated around 28% of the scholarly literature is Open Access {cite}`Piwowar2018stateofOA`and in 2020 more outputs were published through Open Access channels than traditional subscription channels globally {cite}`Dimensions2021openaccesssurpasses`.
+
+(rr-open-access-routes)=
+## Routes for Open Access
```

Comment:
Might benefit from a short paragraph here acknowledging there's lots of different ways to achieve open access depending on your resources, the norms in your field or discipline, and funder restrictions

Reply:
Have done, let me know what you think?